tokio/src/util/mod.rs

```diff
@@ -6,14 +6,8 @@ cfg_io_driver! {
 #[cfg(feature = "rt")]
 pub(crate) mod atomic_cell;
 
-cfg_has_atomic_u64! {
-    #[cfg(any(feature = "signal", all(unix, feature = "process")))]
-    pub(crate) mod once_cell;
-}
-cfg_not_has_atomic_u64! {
-    #[cfg(any(feature = "rt", feature = "signal", all(unix, feature = "process")))]
-    pub(crate) mod once_cell;
-}
+#[cfg(any(feature = "rt", feature = "signal", feature = "process"))]
+pub(crate) mod once_cell;
```

Comment:
do we use the `once_cell` module outside of `cfg_not_has_const_mutex_new`? if we don't, we could just add the cfg as well, rather than allowing dead code in the module. not a blocker.

Reply:
Yes. The signal module uses it unconditionally.